netlogo-gui/src/test/app/infotab/InfoFormatterTests.scala

```diff
@@ -104,13 +105,13 @@ class InfoFormatterTests extends FunSuite {
 
   // in contrast, here no <pre> because no blank line
   test("fenced code blocks 2") {
-    assertResult("""|<p>foo<br/><code>
-              |   bar
-              |  baz
-              |qux
-              |</code><br/>oof</p>""".stripMargin) {
+    assertResult("""|<p>foo<br/><code class="text">
```

Comment:
I agree. It's fine for info tabs to require newlines between sections. The test can (and should) be renamed.